network-api/networkapi/wagtailpages/pagemodels/publications/article.py

```diff
@@ -23,6 +27,25 @@
 ]
 
 
+class ArticleAuthors(Orderable):
+    """This allows us to select one or more blog authors from Snippets."""
+
+    page = ParentalKey("wagtailpages.ArticlePage", related_name="authors")
+    author = models.ForeignKey(
+        BlogAuthor,
+        on_delete=models.SET_NULL,
+        null=True,
+        blank=False
```

Comment:
Authors are optional

Reply:
Yep, these are optional. It's set by the `min_num` value with `null=True`. Blank just means if an author Orderable "Add Author" button is clicked, then make sure it's filled out instead of providing an empty value.